packages/contracts-bedrock/src/Safe/LivenessModule.sol

```diff
@@ -14,6 +14,12 @@ import { ISemver } from "src/universal/ISemver.sol";
 ///         If the number of owners falls below the minimum number of owners, the ownership of the
 ///         safe will be transferred to the fallback owner.
 contract LivenessModule is ISemver {
+    /// @notice Emitted when an owner is removed due to insufficient liveness
+    event RemovedOwner(address indexed owner);
+
+    /// @notice Emitted when the fallback owner takes ownership
+    event FallbackOwnerTakesOwnership();
```

Comment:
Nit on tense, `RemovedOwner` is past tense and `FallbackOwnerTakesOwnership` is present tense

Reply:
OK, both are past tense now.